dev-tools/scripts/releaseWizard.yaml

```diff
@@ -411,7 +410,7 @@ groups:
         cmd: git pull
         stdout: true
       - !Command
-        cmd: ant clean precommit
+        cmd: ./gradlew clean check -x test
```

Comment:
Will the `./` syntax work on Windows?

Reply:
It works in PowerShell but not Command Prompt. I don't know how important this distinction is.

Reply:
You can definere a variable like we do for rename command, that used `gradlew.bat` for win?

Reply:
Works great!